multibotconfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "latitude" : latitude,
+  "longitude" : longitude,
+  "credentials" : {
+    "type" : "google-auto", OR "type" : "ptc", (choose wisely)
+    "username" : "username",
+    "password" : "password"
+  },
+  "speed" : 2.8,
+  "followStreets" : false,
+  "dropItems" : true,
+  "uselessItems" : {
+    "ITEM_REVIVE" : 20,
+    "ITEM_MAX_REVIVE" : 10,
+    "ITEM_POTION" : 0,
+    "ITEM_SUPER_POTION" : 30,
+    "ITEM_HYPER_POTION" : 50,
+    "ITEM_MAX_POTION" : 50,
+    "ITEM_POKE_BALL" : 40,
+    "ITEM_GREAT_BALL" : 50,
+    "ITEM_ULTRA_BALL" : 50,
+    "ITEM_MASTER_BALL" : 10,
+    "ITEM_RAZZ_BERRY" : 40,
+    "ITEM_LUCKY_EGG" : -1,
+    "ITEM_INCENSE_ORDINARY" : -1,
+    "ITEM_TROY_DISK" : -1
+  },
+  "profileUpdateTimer" : 60,
+  "timerWalkToStartPokestop" : -1,
+  "randomNextPokestopSelection" : 5,
+  "desiredCatchProbability" : 0.4,
+  "desiredCatchProbabilityUnwanted" : 0.0,
+  "autotransfer" : true,
+  "keepPokemonAmount" : 1,
+  "maxPokemonAmount" : -1,
+  "displayKeepalive" : true,
+  "displayPokestopName" : false,
+  "displayPokestopRewards" : true,
+  "displayPokemonCatchRewards" : true,
+  "displayIfPokemonFromLure" : true,
+  "lootPokestop" : true,
+  "catchPokemon" : true,
+  "autoFillIncubator" : true,
+  "sortByIv" : false,
+  "alwaysCurve" : false,
+  "neverUseBerries" : true,
+  "allowLeaveStartArea" : false,
+  "spawnRadius" : -1,
+  "banSpinCount" : 0,
+  "transferCpThreshold" : 400,
+  "transferIvThreshold" : 80,
+  "ignoredPokemon" : [ "EEVEE", "MEWTWO", "CHARMANDER" ],
+  "obligatoryTransfer" : [ "DODUO", "RATTATA", "CATERPIE", "PIDGEY" ],
+  "export" : "",
+  "guiPortSocket" : 8001,
+  "initialMapSize" : 3,
+  "version" : "0.5.0-alpha4"
```

Comment:
should be removed I guess, as this is should be generated at runtime. should probably also remove it in the `data class Settings` annotation. @apottere ?